webapp/src/components/training/Trainer.vue

```diff
@@ -59,7 +59,7 @@
             <!-- Display an activity indicator depending on the training status -->
             <div class="min-h-9">
               <div v-if="roundStatus === 'Waiting for more participants' ||
-                roundStatus === 'Retrieving peers\' information'"
+                roundStatus === 'Establishing peer-to-peer connections'"
```

Comment:
checking on such strings is quite error-prone (it will silently missbehave when changing the formatted text). updating `roundStatus` to `ref<[RoundStatus, string]>` helps by having both a safe value to check (`RoundStatus`) and a formatted string to print